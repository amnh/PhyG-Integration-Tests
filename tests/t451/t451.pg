--input t5.tre, swap(annealing, steps:3), inputting fasta and binary files
set(seed:1634561640)
read(fasta:"chel.fas", "chel_binary.fas")
read(newick:"../t5/t5_nocollapse.tre")
swap(annealing, steps:3)
report("t451.csv", data, overwrite)
report("t451.tre", newick, overwrite)