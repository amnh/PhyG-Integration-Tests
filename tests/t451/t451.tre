(Alentus:69.0,(Aportus:25.0,(Vonones:21.0,(Amblypygid:16.5,((Rhiphicephalus:34.5,(Thermobius:30.0,(Americhernus:54.5,Artemia:24.5)HTU19:18.0)HTU24:14.0)HTU23:11.0,(Mastigoproctus:21.0,(((Centruoides:18.5,(Hadrurus:19.0,Pauroctonus:13.0)HTU27:18.0)HTU25:10.5,(Chanbria:29.0,Limulus:18.0)HTU32:8.5)HTU22:10.0,(Gea:38.0,(Hypochilus:10.5,Thelichoris:13.5)HTU26:8.0)HTU31:9.5)HTU29:5.0)HTU28:7.0)HTU20:9.5)HTU30:8.5)HTU21:19.0)HTU18:61.5)HTU17[505.0];
(Alentus:69.0,(Aportus:25.0,(Vonones:21.0,(Amblypygid:16.5,((Mastigoproctus:21.0,(((Centruoides:18.5,(Hadrurus:19.0,Pauroctonus:13.0)HTU30:18.0)HTU25:10.5,(Chanbria:29.0,Limulus:18.0)HTU32:8.5)HTU20:10.0,(Gea:38.0,(Hypochilus:10.5,Thelichoris:13.5)HTU24:8.0)HTU31:9.5)HTU21:5.0)HTU23:7.0,(Rhiphicephalus:34.5,(Thermobius:30.0,(Americhernus:54.5,Artemia:24.5)HTU26:18.0)HTU27:14.0)HTU28:11.0)HTU29:9.5)HTU22:8.5)HTU19:19.0)HTU18:61.5)HTU17[505.0];
(Alentus:18.5,(Aportus:22.5,(Rhiphicephalus:29.0,(Thermobius:33.5,((Amblypygid:20.5,Vonones:25.5)HTU20:6.5,((Americhernus:49.5,Artemia:37.5)HTU22:19.5,((Gea:36.0,(Hypochilus:16.0,Thelichoris:18.0)HTU29:7.5)HTU27:9.0,(Mastigoproctus:23.0,((Centruoides:23.0,(Hadrurus:23.0,Pauroctonus:16.0)HTU26:17.5)HTU25:10.0,(Chanbria:34.0,Limulus:22.0)HTU32:8.5)HTU30:7.5)HTU28:7.0)HTU23:7.5)HTU31:9.0)HTU24:11.0)HTU19:14.5)HTU21:21.5)HTU18:25.0)HTU17[505.0];
(Alentus:26.5,(Aportus:13.5,(Thermobius:30.5,(Rhiphicephalus:32.0,((Amblypygid:16.0,Vonones:22.0)HTU20:7.5,((Americhernus:47.5,Artemia:33.5)HTU22:22.5,((Gea:35.0,(Hypochilus:11.0,Thelichoris:14.0)HTU29:7.5)HTU27:9.5,(Mastigoproctus:19.5,((Centruoides:18.5,(Hadrurus:19.5,Pauroctonus:11.5)HTU26:19.5)HTU25:10.5,(Chanbria:28.5,Limulus:18.5)HTU32:8.0)HTU30:8.0)HTU28:7.0)HTU23:7.5)HTU31:10.0)HTU24:12.5)HTU19:9.0)HTU21:34.5)HTU18:25.0)HTU17[505.0];
(Alentus:26.5,(Aportus:13.5,(Thermobius:30.5,(Rhiphicephalus:32.0,((Amblypygid:16.0,Vonones:22.0)HTU26:7.5,((Americhernus:47.5,Artemia:33.5)HTU20:22.5,((Gea:35.0,(Hypochilus:11.0,Thelichoris:14.0)HTU21:7.5)HTU19:9.5,(Mastigoproctus:19.5,((Chanbria:28.5,Limulus:18.5)HTU29:8.0,(Centruoides:18.5,(Hadrurus:19.5,Pauroctonus:11.5)HTU32:19.5)HTU31:10.5)HTU24:8.0)HTU28:7.0)HTU22:7.5)HTU27:10.0)HTU30:12.5)HTU25:9.0)HTU23:34.5)HTU18:25.0)HTU17[505.0];
(Alentus:18.5,(Aportus:22.5,(Rhiphicephalus:29.0,(Thermobius:33.5,((Amblypygid:20.5,Vonones:25.5)HTU26:6.5,((Americhernus:49.5,Artemia:37.5)HTU20:19.5,((Gea:36.0,(Hypochilus:16.0,Thelichoris:18.0)HTU21:7.5)HTU19:9.0,(Mastigoproctus:23.0,((Chanbria:34.0,Limulus:22.0)HTU29:8.5,(Centruoides:23.0,(Hadrurus:23.0,Pauroctonus:16.0)HTU32:17.5)HTU31:10.0)HTU24:7.5)HTU28:7.0)HTU22:7.5)HTU27:9.0)HTU30:11.0)HTU23:14.5)HTU25:21.5)HTU18:25.0)HTU17[505.0];
(Alentus:26.5,(Aportus:13.5,(Thermobius:30.5,(Rhiphicephalus:32.0,((Amblypygid:16.0,Vonones:22.0)HTU20:7.5,(((Gea:35.0,(Hypochilus:11.0,Thelichoris:14.0)HTU32:7.5)HTU22:9.5,(Mastigoproctus:19.5,((Chanbria:28.5,Limulus:18.5)HTU26:8.0,(Centruoides:18.5,(Hadrurus:19.5,Pauroctonus:11.5)HTU28:19.5)HTU27:10.5)HTU24:8.0)HTU25:7.0)HTU21:7.5,(Americhernus:47.5,Artemia:33.5)HTU31:22.5)HTU30:10.0)HTU19:12.5)HTU23:9.0)HTU29:34.5)HTU18:25.0)HTU17[505.0];
(Alentus:18.5,(Aportus:22.5,(Rhiphicephalus:29.0,(Thermobius:33.5,((Amblypygid:20.5,Vonones:25.5)HTU20:6.5,(((Gea:36.0,(Hypochilus:16.0,Thelichoris:18.0)HTU32:7.5)HTU22:9.0,(Mastigoproctus:23.0,((Chanbria:34.0,Limulus:22.0)HTU26:8.5,(Centruoides:23.0,(Hadrurus:23.0,Pauroctonus:16.0)HTU28:17.5)HTU27:10.0)HTU24:7.5)HTU25:7.0)HTU21:7.5,(Americhernus:49.5,Artemia:37.5)HTU31:19.5)HTU30:9.0)HTU19:11.0)HTU29:14.5)HTU23:21.5)HTU18:25.0)HTU17[505.0];
(Alentus:18.5,(Aportus:22.5,(Rhiphicephalus:29.0,(Thermobius:33.5,((((Gea:36.0,(Hypochilus:16.0,Thelichoris:18.0)HTU28:7.5)HTU21:9.0,(Mastigoproctus:23.0,((Centruoides:23.0,(Hadrurus:23.0,Pauroctonus:16.0)HTU25:17.5)HTU24:10.0,(Chanbria:34.0,Limulus:22.0)HTU26:8.5)HTU23:7.5)HTU27:7.0)HTU22:7.5,(Americhernus:49.5,Artemia:37.5)HTU31:19.5)HTU20:9.0,(Amblypygid:20.5,Vonones:25.5)HTU29:6.5)HTU19:11.0)HTU32:14.5)HTU30:21.5)HTU18:25.0)HTU17[505.0];
(Alentus:26.5,(Aportus:13.5,(Thermobius:30.5,(Rhiphicephalus:32.0,((((Gea:35.0,(Hypochilus:11.0,Thelichoris:14.0)HTU28:7.5)HTU21:9.5,(Mastigoproctus:19.5,((Centruoides:18.5,(Hadrurus:19.5,Pauroctonus:11.5)HTU25:19.5)HTU24:10.5,(Chanbria:28.5,Limulus:18.5)HTU26:8.0)HTU23:8.0)HTU27:7.0)HTU22:7.5,(Americhernus:47.5,Artemia:33.5)HTU31:22.5)HTU20:10.0,(Amblypygid:16.0,Vonones:22.0)HTU29:7.5)HTU19:12.5)HTU30:9.0)HTU32:34.5)HTU18:25.0)HTU17[505.0];
