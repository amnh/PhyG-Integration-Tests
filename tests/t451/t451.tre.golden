(Alentus:28.0,(Aportus:4.0,(Vonones:9.0,(Amblypygid:7.0,((Rhiphicephalus:19.0,(Artemia:24.0,Thermobius:10.0)HTU32:6.0)HTU20:6.0,(Mastigoproctus:12.0,(((Centruoides:10.0,(Hadrurus:8.0,Pauroctonus:3.0)HTU25:13.0)HTU24:6.0,(Chanbria:21.0,Limulus:10.0)HTU26:4.0)HTU23:4.0,(Thelichoris:4.0,(Hypochilus:1.0,(Americhernus:33.0,Gea:9.0)HTU31:11.0)HTU30:1.0)HTU29:5.0)HTU27:3.0)HTU22:3.0)HTU21:5.0)HTU28:3.0)HTU19:11.0)HTU18:27.0)HTU17[507.0];
(Alentus:28.0,(Aportus:4.0,(Vonones:9.0,(Amblypygid:7.0,((Rhiphicephalus:19.0,(Artemia:24.0,Thermobius:10.0)HTU32:6.0)HTU20:6.0,(Mastigoproctus:12.0,(((Centruoides:10.0,(Hadrurus:8.0,Pauroctonus:3.0)HTU25:13.0)HTU24:6.0,(Chanbria:21.0,Limulus:10.0)HTU26:4.0)HTU23:4.0,(Hypochilus:1.0,(Thelichoris:2.0,(Americhernus:31.0,Gea:11.0)HTU31:10.0)HTU30:1.0)HTU29:5.0)HTU27:3.0)HTU22:3.0)HTU21:5.0)HTU28:3.0)HTU19:11.0)HTU18:27.0)HTU17[507.0];
